sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -13502,16 +12768,16 @@
                                 "value": ""
                               }
                             ],
-                            "content": "CSGT\n <span\n class=\"formio-custom-tooltip\">\n   <span class=\"tooltip-text\">\n    Canada Student Grant for Full-time Top-up\n    </span>\n</span>",
+                            "content": "Canada Student Grant for Students with Dependants (CSG_PTDEP)\n<span\n class=\"formio-custom-tooltip\">\n  <span class=\"tooltip-text\">\n    Please click <a\n            href=\"https://studentaidbc.ca/explore/grants-scholarships/canada-student-grant-students-dependants\"\n            rel=\"noopener\"\n            target=\"_blank\"\n            >here</a> for more information about this grant.\n    </span>\n</span>",
```

Comment:
Figma does not have the acronyms appended (e.g. CSG_PTDEP) and I would rely more in Figma unless there is a business AC that would indicate it.
Should the acronyms appended?
If yes, this one is not following the acronym pattern and it should be `CSG-PTDEP`

Reply:
I had talked to Jeremy about it and he confirmed that the acronyms should be appended as the ACs.
I'm fixing the acronym.